src/utils/index.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Takes a unix timestamp and converts it to a locale string.
+ * @param {number} unixTimestamp A unix timestamp, e.g. 1617145200000
+ * @returns {string} A formatted locale string, e.g. "April 3, 2:30 PM"
+ */
+export const formatUnixToLocaleString = unixTimestamp => {
+  return new Date(unixTimestamp).toLocaleString('en-US', {
+    month: 'long',
+    day: 'numeric',
+    hour: 'numeric',
+    year: 'numeric',
+    minute: '2-digit',
+    timeZone: 'CST',
```

Comment:
Technically this would be CDT right now... Thoughts on how we should handle this? We can figure out whether it's CST or CDT programmatically based on the day.

Reply:
We can likely just check the offset to UTC. I can add that logic in for sure.

Reply:
We're in luck. The events that come back from the Meetup API include a Unix timestamp and a UTC offset, so we can omit the `timeZone` option when converting to locale string and just do a simple parsing of `time - utc_offset`.
![time_and_offset](https://user-images.githubusercontent.com/23620564/116799124-d3907c80-aabb-11eb-9a27-f1a0c6391a8c.png)

Reply:
That's very convenient! Good catch! Thanks for updating it.